action.yaml

```
@@ -89,12 +93,21 @@ runs:
           const repo = "${{ steps.get-repo.outputs.result }}"
 
           if (${{ steps.get-id.outputs.result != -1 && inputs.post-to-issue }}) {
+            if (${{ inputs.append-to-issue }}) {
+              github.rest.issues.createComment({
+                owner: owner,
+                repo: repo,
+                issue_number: "${{ steps.get-id.outputs.results }}",
+                boxy: issue_body
```

```suggestion
                body: issue_body
```
:D https://github.com/assignUser/simstudy/runs/5006623052?check_suite_focus=true#step:3:1267